CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# 4.3.1 (2021-03-17)
+- [NEW] Add migration guide to new SDK.
```

Comment:
Should we add beta?
```suggestion
- [NEW] Add migration guide to new beta SDK.
```

Reply:
The migration guide doesn't mention "beta" so I'm not sure about that. I can change the wording to "to the newly supported cloudant-node-sdk (package: @ibm-cloud/cloudant) " to match the guide.

Reply:
Ok that sounds good to me.